src/test/apisix/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
KS UI reports the test job is failed. We need make it disabled by default to avoid the failure, like 
https://github.com/kubesphere/helm-charts/blob/2afc50c85741a5f781a1cb93d5f8492ca67b99c2/src/test/etcd/templates/tests/test-connection.yaml#L1

Reply:
https://github.com/kubesphere/helm-charts/blob/2afc50c85741a5f781a1cb93d5f8492ca67b99c2/src/test/etcd/values.yaml#L39-L40

Reply:
done